backend/controllers/Auth/LoginController.js

```diff
@@ -0,0 +1,22 @@
+class LoginController {
+    constructor(userService, httpStatusCodes) {
+        this.userService = userService;
+        this.httpStatusCodes = httpStatusCodes;
+    }
+
+    async invoke(req, res) {
+        const { email, password } = req.body;
+
+        const user = await this.userService.login(email, password);
+
+        if (!user) {
+            return res.sendStatus(this.httpStatusCodes.UNAUTHORIZED);
+        }
+
+        req.session.user = user;
+
+        return res.send(user);
```

Comment:
without deleting password, you are sending encrypted password to the hacker, is this a deliberate procedure?